conduit/pschema/schema.go

```diff
@@ -32,16 +34,17 @@ const (
 )
 
 type CreateRequest struct {
-	Name  string
-	Type  Type
-	Bytes []byte
+	Subject string
+	Type    Type
```

Comment:
We should reuse the [type](https://github.com/ConduitIO/conduit-commons/blob/0daeaaf8c952627b6f1ccd85c4904f02a75b1b9a/schema/schema.go#L19) we have in the commons package.